components/calendar/nz-calendar-header.component.html

```diff
@@ -0,0 +1,14 @@
+<nz-select class="ant-fullcalendar--year-select" [nzSize]="size"
```

Comment:
the className should be changed when the nzFullscreen change

Reply:
This class is static in current react implementation

Reply:
className should be `ant-fullcalendar-year-select`